src/main/java/executor/service/exceptions/StepExecutionException.java

```diff
@@ -0,0 +1,20 @@
+package executor.service.exceptions;
+
+public class StepExecutionException extends RuntimeException {
```

Comment:
Wrapping a RuntimeException in a custom Exception makes sense if you want to add additional information or context to the exception. Alternatively, add a log.error().

Reply:
Are you shure that there are no context in this exception?